mvp/src/main/java/com/distillery/android/blueprints/mvp/architecture/BaseContractView.kt

```diff
@@ -0,0 +1,7 @@
+package com.distillery.android.blueprints.mvp.architecture
+
+interface BaseContractView {
+    fun startLoading()
+    fun endLoading()
```

Comment:
The view itself doesn't load anything so these names aren't quite descriptive. What implementations of these methods are supposed to do? If it's showing/hiding a kind of a progress indicator when names can be something like:
```suggestion
    fun showProgressIndicator()
    fun hideProgressINdicator()
```

Reply:
These methods are responsible not only for showing the progressbar, but also for hiding and showing the main content. Therefore, the name Loading will be more appropriate, I think.